src/main/java/core/basesyntax/service/FruitsFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FruitsFromFile {
+    List<String> getFruits(List<FruitTransaction> fruitTransactions);
```

Comment:
The method signature `getFruits` is not descriptive of its functionality. It should be named to clearly indicate its purpose. If it's converting transactions to strings, a name like `convertTransactionsToStrings` or `processTransactions` might be more appropriate.